MarkBind/markbind
# Upgrade MarkBind dependencies

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Other, please explain: DevOps



Part of #461 



**What is the rationale for this request?**
There are currently 5 vulnerabilities within MarkBind. Since the next release is a minor increment due to breaking changes in #864 , Let's upgrade our dependencies as well.

**What changes did you make? (Give an overview)**

``` Markdown
* fastmatter: 2.1.0 → 2.1.1
* eslint: 5.15.3 → 5.16.0
* jest: 24.5.0 → 24.8.0
* stylelint: 9.10.1 -> 10.0.1
* stylelint-config-standard: 18.2.0 → 18.3.0
```

Note that these required packages were manually edited to use `js-yaml@3.13.1` |
--- |
Eslint https://github.com/MarkBind/markbind/blob/e7645385180ab395d1d2c794fd1ae766026ae741/package-lock.json#L2255 |
Fastmatter: https://github.com/MarkBind/markbind/blob/e7645385180ab395d1d2c794fd1ae766026ae741/package-lock.json#L2768

**Is there anything you'd like reviewers to focus on?**
- Commit message

**Testing instructions:**
- Run `npm audit` (on npm@latest) and ensure 0 vulnerabilities found.
- Open Netlify Preview and see nothing breaks.
- Build other large MarkBind websites E.G [CS2103](https://github.com/nus-cs2103/website-base) and check that nothing breaks.
